src/redux/news/tests/news.saga.test.js

```diff
@@ -0,0 +1,100 @@
+import { expectSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+import { throwError } from 'redux-saga-test-plan/providers';
+import { handleNewsLoad, handleArticleLoad } from '../news.sagas';
+import getItems from '../../../utils/client';
+import { SET_NEWS, SET_NEWS_ARTICLE, SET_LOADING } from '../news.types';
+import { SET_ERROR } from '../../error/error.types';
+
+describe('get news saga', () => {
+  it('fetches news', () => {
+    const newsExample = {
+      data: {
+        getAllNews: {
+          news: [
+            {
+              author: {
+                image: 'someImage.jpeg',
+                name: 'Author Name'
+              },
+              date: '154568794567',
+              title: 'Test title',
+              text: 'Test text'
+            },
+            {
+              author: {
+                image: 'otherImage.jpeg',
+                name: 'Second Author Name'
+              },
+              date: '154568794567',
+              title: 'Second Test title',
+              text: 'Second Test text'
+            }
+          ]
+        }
+      }
+    };
+
+    return expectSaga(handleNewsLoad)
+      .provide([[matchers.call.fn(getItems), newsExample]])
+      .put({ type: SET_LOADING, payload: true })
+      .put({
+        type: SET_NEWS,
+        payload: newsExample.data.getAllNews
+      })
+      .put({ type: SET_LOADING, payload: false })
+      .run();
+  });
+});
+
+it('handles errors', () => {
+  const e = new Error('news not found');
```

Comment:
Please, remove this redundant variable and inline creation in the action creator

Reply:
I simulate error here